sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -144,7 +144,7 @@ await using (var client = new EventHubClient(connectionString, scope.EventHubNam
                     Assert.That(properties, Is.Not.Null, "A set of properties should have been returned.");
                     Assert.That(properties.Path, Is.EqualTo(scope.EventHubName), "The property Event Hub name should match the scope.");
                     Assert.That(properties.PartitionIds.Length, Is.EqualTo(partitionCount), "The properties should have the requested number of partitions.");
-                    Assert.That(properties.CreatedAtUtc, Is.EqualTo(DateTime.UtcNow).Within(TimeSpan.FromSeconds(10)), "The Event Hub should have been created just about now.");
+                    Assert.That(properties.CreatedAt, Is.EqualTo(DateTimeOffset.Now).Within(TimeSpan.FromSeconds(10)), "The Event Hub should have been created just about now.");
```

Comment:
These should be `DateTimeOffset.UtcNow.`    Not a huge deal in the tests, but `UtcNow` = the UTC offset (nothing) where `Now` uses the local timezone offset.  

[reference](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.now?view=netframework-4.8)

Reply:
Done.